include/irr/asset/IMeshPacker.h

```diff
@@ -48,13 +55,20 @@ namespace meshPackerUtil
     {
         uint32_t mdiParameterOffset; // add to `CCPUMeshPacker::getMultiDrawIndirectBuffer()->getPointer() to get `DrawElementsIndirectCommand_t` address
         uint32_t mdiParameterCount;
+
+        inline bool isValid()
+        {
+            return this->mdiParameterOffset != core::GeneralpurposeAddressAllocator<uint32_t>::invalid_address;
+        }
     };
 
     
 }
 
 using namespace meshPackerUtil;
 
+//TODO: allow mesh buffers with only per instance attributes
+
 template <typename MeshBufferType, typename MDIStructType>
```

Comment:
nothing in this class depends on `MeshBufferType` so the template parameter is not necessary.